src/Forms/ConfigurationForm.cs

```diff
@@ -16,14 +16,26 @@ public ConfigurationForm(AzureMigrateExportMainForm obj)
         {
             InitializeComponent();
             mainFormObj = obj;
+
+            // Set the DropDownStyle to DropDownList to disable text entry
+            ModuleComboBox.DropDownStyle = ComboBoxStyle.DropDownList;
         }
 
         #region Set Default Values
         public void SetDefaultConfigurationValues()
         {
+            if (mainFormObj.isImportDiscovery())
+            {
+                HyperVCheckBox.Enabled = false;
+                VMwareCheckBox.Enabled = false;
+                PhysicalCheckBox.Enabled = false;
+            }
+            else
+            {
             HyperVCheckBox.Checked = true;
             VMwareCheckBox.Checked = true;
             PhysicalCheckBox.Checked = true;

```

Comment:
This entire logic should not be required.

The SetDefaultConfigurationValues is called only once from the Azure Migrate Export main form.

So, at that time, we can set "HYPERV" "PHYSICAL" "VMWARE" enabled and import as disabled.

Then you need to add "Checkbox_Checked()" functions, for all 4 checkboxed, for appliance check boxes, you disable and uncheck import and for import checkbox you should disable and uncheck import.